icu4c/source/test/Makefile.in

```diff
@@ -86,6 +85,13 @@ all-recursive install-recursive clean-recursive distclean-recursive dist-recursi
 	  $(MAKE) "$$target-local" || exit; \
 	fi
 
+check-recursive: intltest $(IOTEST) cintltst $(LETEST) $(FUZZER)
+	$(MAKE) check-local || exit;
+
+intltest $(IOTEST) cintltst $(LETEST) $(FUZZER):
+	(cd testdata; $(MAKE) -j all) || exit;
+	(cd $@ && $(MAKE) -j -O check) || exit;
```

Comment:
same as above, should be able to use a variable. We shouldn't have the paths in multiple places